style-shelter.js

```diff
@@ -118,6 +118,14 @@ export default {
             return this._dict.get(url);
         } else {
             const sheet = new CSSStyleSheet();
+
+            const documentSheet = Array(...document.styleSheets).find(styleSheet => styleSheet.href.includes(url));
+            if (documentSheet) {
+              sheet.replace(Array(...documentSheet.rules).map(rule => rule.cssText).join(' '));
```

Comment:
You can't use non-constructed stylesheets with "adoptedStylesheets" (aka: `whatever.adoptedStylesheets = [documentStylesheet, ...whatever.adoptedStylesheets]`) ... idk why.
BUT, we can just re-create the contents of the entire stylesheet and use that as our constructed stylesheet.